README.md

```diff
@@ -9,6 +9,196 @@
 
 ## Visual Schema
 
+### Report Model
+#### REQUEST
+```
+User clicks submit on survey
+# POST http://localhost:5000/api/v1/report
+{
+  salary: 55000,
+  zipcode: 11111,
+  credit: 695,
+  monthly_debt: 1100,
+  downpayment_savings: 10000,
+  downpayment_percentage: 10,
+  rent: 1800,
+  goal_principal: 0
+}
+```
+
+#### RESPONSE
+```
+# The following is the json return to the POST from above
+{
+  report: {
+    location: {
+      zip_code: 11111,
+      location: "Anywhere, CO",
+    },
+    principal: {
+      based_on_rent: 350000,
+      goal_principal: 0
+    },
+    monthly: {
+      monthly_principal: 1400,
+      estimated_true_monthly: 1940,
+      add_ons: {
+        home_insurance: 110,
+        property_tax: 105,
+        hoa: 75,
+        pmi: 250
+      }
+    },
+    downpayment: {
+      down_payment_percentage_selected: 10,
+      down_payment_saved: 10000,
+      down_payment_percent_saved: 2.9,
+      ten_year_plan: {
+        one: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2022
+        },
+        two: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2023
+        },
+        three: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2024
+        },
+        four: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2025
+        },
+        five: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2026
+        },
+        six: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2027
+        },
+        seven: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2028
+        },
+        eight: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2029
+        },
+        nine: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2030
+        },
+        ten: {
+          monthly_savings: 100,
+          goal_end_date: 12/03/2031
+        }
+      }
+    }
+  }
+}
+```
+
+### Education Model
+#### REQUEST
+```
+User clicks submit on survey
+# GET http://localhost:5000/api/v1/education
```

Comment:
I think we will actually be making this request on app load, or somewhere higher up in the process. 

I think "User clicks submit on survey" could be replaced with "user loads the application"

Reply:
That's cool!  Does that help with load time?